### PR TITLE
About and project section

### PR DESCRIPTION
**Pull request by EidHachem**
Add About and Projects sections with a mobile responsive design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,6 +26,119 @@ <h1 class="greeting">Eid Hachem</h1>
...
+            <li class="social-item"><img src="images/social/twitter.svg" alt=""></li>
+            <li class="social-item gmail"><img src="images/social/mail.svg" alt="Gmail"></li>
+            <li class="social-item-work social-item-work-right second-arrow"><img src="images/home/Group.svg" alt="Arrow down"></li>
+            <li class="social-item second-work"><p class="social-item-p-right">Work</p></li>
+        </ul>
+    </nav>
+    <section class="work">
+        <h3 class="work-header">My recent work</h3>
+        <img src="images/home/tall-line.svg" alt="line" class="work-line">
+    </section>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please the container of the section is a section tag. Also, the container has only one (1) `<section>` tag. Make sure that your code looks more semantic. 👍 
I provided the [Semantic HTML](https://www.w3docs.com/learn-html/semantic-elements-in-html5.html) Link here

File: index.html [hunk 2]
```diff
@@ -26,6 +26,119 @@ <h1 class="greeting">Eid Hachem</h1>
...
+            <ul class="project-langs">
+                <li class="langs-used css">css</li>
+                <li><img src="images/home/split.svg" alt="vector" class="lang-split"></li>
+                <li class="langs-used html">html</li>
+                <li><img src="images/home/split.svg" alt="vector" class="lang-split"></li>
+                <li class="langs-used bootstrap">bootstrap</li>
+                <li><img src="images/home/split.svg" alt="vector" class="lang-split"></li>
+                <li class="langs-used rubu">Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
```
Comment (OLIPLICHE) [inline index.html]: - [x] Please make sure the button `See project`  follows the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)) 👍

File: index.html [hunk 3]
```diff
@@ -26,6 +26,119 @@ <h1 class="greeting">Eid Hachem</h1>
...
+            <li class="langs-used html">html</li>
+            <li><img src="images/home/split.svg" alt="" class="lang-split"></li>
+            <li class="langs-used bootstrap">bootstrap</li>
+            <li><img src="images/home/split.svg" alt="" class="lang-split"></li>
+            <li class="langs-used rubu">Ruby</li>
+        </ul>
+        <button type="button" class="project-button">See Project</button>
+    </div>
+    <img class="project-image" src="images/home/Placeholder.svg" alt="">
+</section>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please the Section `Contact Us` is missing. Make sure to add it.
- [ ] Original design
h1

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @EidHachem , 

Good job so far!
There are some issues that you still need to work on to get your project approved, but you are almost there!


## Highlights
- [ ] Linter result is green ✔️
- [ ] Commit messages are meaningful✔️
- [ ] node_modules folder is in the .gitignore file and not in the repo✔️
- [ ] README file is descriptive✔️


## Required Changes ♻️

- [x] **Original Design**: 
x4
x2

x3


- [x] **Your Design**:
x6
x5

- [ ] Please, Make sure to build your project from scratch. Follow the[ Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)
- [ ] Please do not ignore the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).

- [ ] **CHECK THE COMMENTS UNDER THE REVIEW**⬇.. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider**:
- Pull Request should have a descriptive title telling you what changes are inside.✔️
- Pull Request should have a short summary describing introduced changes
   in more detail✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (EidHachem) [reply index.html]: Hello @OLIPLICHE 
Thanks for the review, I'm working on the required changes.
The Contact us section, is not included in this project, we were asked to build works and about sections only.
Please advise if I should add it.

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢

Hi @EidHachem, 

Good job implementing the changes the previous reviewer asked for.
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉

ClappingLeonardoDicaprioGIF


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**